Makefile

```diff
@@ -1,5 +1,5 @@
 NAME          = rmt-server
-VERSION       = 1.0.4
+VERSION       = $(shell ruby -e 'require "./lib/rmt.rb"; print RMT::VERSION')
```

Comment:
when I execute the command I get `shell: command not found`
could `$(ruby -e 'require "./lib/rmt.rb"; print RMT::VERSION')`  also work?

Reply:
How are you executing the command?

Reply:
sry I did run in via my CLI to test.
packaging works fore me so never mind